### DISCUSSION TITLE
How do animated Backgrounds work?

### DISCUSSION THREAD

**Discussion by Squidward578** [TrianglyRU/OrbinautFramework#47] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bb8f0a78-dd55-4900-9bdc-76fae681ed8e)
whenever i attempt to use an animated background, it always pulls frames from other animations
![image](https://github.com/user-attachments/assets/29d47708-b2dd-43c4-9a9c-973332475350)
![image](https://github.com/user-attachments/assets/12d6ffac-aa8d-4497-8603-d6400d9c3d59)
![image](https://github.com/user-attachments/assets/adee673a-79a8-4adf-b8a0-b0817d0d06d6)

**Comment by TrianglyRU** [comment, marked as answer]
The background setup is correct. The issue is, your background sprite has empty space around it, and it is assigned to the texture group which has "Automatically Crop" setting checked. Either assign the sprite to a texture group without this setting checked, or crop the sprite to remove empty space

**Comment by Squidward578** [comment]
thanks for the help, it works now
